build.gradle

```
@@ -31,10 +31,14 @@ dependencies {
 	// jwt
 	implementation 'io.jsonwebtoken:jjwt:0.9.1'
 
+	// redis
+	implementation('org.springframework.boot:spring-boot-starter-data-redis')
+
 	testImplementation 'io.rest-assured:rest-assured:3.3.0'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 
 	runtimeOnly 'com.h2database:h2'
+	implementation("mysql:mysql-connector-java")
```

```suggestion
	implementation 'mysql:mysql-connector-java'
```

다른 설정과 동일하게 통일해보면 어떨까요?